Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyProcessOrder.Codeunit.al

```diff
@@ -150,6 +152,16 @@ codeunit 30166 "Shpfy Process Order"
         OrderEvents.OnAfterCreateSalesHeader(ShopifyOrderHeader, SalesHeader);
     end;
 
+    local procedure UpdatePaymentTerms(var SalesHeader: Record "Sales Header"; PaymentTermsType: Code[20]; PaymentTermsName: Text[50])
+    var
+        ShpfyPaymentTerms: Record "Shpfy Payment Terms";
+    begin
+        ShpfyPaymentTerms.SetRange(Type, PaymentTermsType);
```

Comment:
I'm not sure if I understand here evreything correctly, but shouldn't we have here an additional filter on the Shop Code?

Reply:
I think you are correct, good catch!